### PR TITLE
Update Searching.md to document new double quote and wildcard functionality

### PR DESCRIPTION
**Pull request by ScottPatterson**

### CODE REVIEW

File: Documentation/SequentialDataStore/Searching.md [hunk 1]
```diff
@@ -234,6 +230,34 @@ You can use the ``'*'`` character as a wildcard to specify an incomplete string.
...
+
+When using double quotes, the matching string must include the whole value of the field on the object being searched.  Partial strings will not be matched unless wildcards are used.  For example, if you're searching on a stream that has a description of ``Pump three on unit five``, a query of ``"unit five"`` will not match the description, but a query of ``*"unit five"`` will.
+
+Also, wildcards can be used on the *outside* of the quote operators, but if an asterisk is on the inside of the quotes, it is treated as a string literal rather than a wildcard operator.  For example, you can search for ``"dog food"*`` to find a string that starts with "dog food", but if you search for ``"dog food*"``, it will only match on a value of "dog food*".
+
+ **Query string**     | **Matches field value** | **Does not match field value**	
+------------------ | --------------------------------- | -----------------------------	
+``"pump pressure"`` | pump pressure | pressure <br> pressure pump <br> pump pressure gauge	
+``"pump pressure"*`` | pump pressure <br> pump pressure guage | pressure <br> pressure pump <br> the pump pressure guage
```
Comment (osimarkrogers) [inline Documentation/SequentialDataStore/Searching.md]: change "guage" to "gauge"

File: Documentation/SequentialDataStore/Searching.md [hunk 2]
```diff
@@ -234,6 +230,34 @@ You can use the ``'*'`` character as a wildcard to specify an incomplete string.
...
+``*"pump pressure"`` | pump pressure <br> the pump pressure | pressure <br> pressure pump <br> the pump pressure guage
```
Comment (osimarkrogers) [inline Documentation/SequentialDataStore/Searching.md]: "guage" to "gauge"

**Comment by mckenzielee** [comment]
@osimarkrogers, we need to hold off on merging this PR until the functionality is available in Production, which will probably be in about a month.

**Comment by osimarkrogers** [comment]
Understood.

File: Documentation/SequentialDataStore/Searching.md (hunk 2 above)
Comment (ScottPatterson) [reply Documentation/SequentialDataStore/Searching.md]: Fixed in commit f346ef5c8114b5750bb91111ac51549f7eabbdd2

File: Documentation/SequentialDataStore/Searching.md (hunk 1 above)
Comment (ScottPatterson) [reply Documentation/SequentialDataStore/Searching.md]: Fixed in commit f346ef5c8114b5750bb91111ac51549f7eabbdd2

File: Documentation/SequentialDataStore/Searching.md [hunk 3]
```diff
@@ -211,18 +207,18 @@ You can also qualify which fields are searched by using the following syntax:
...
 
 **Query string**     | **Matches field value** | **Does not match field value**
 ------------------ | --------------------------------- | -----------------------------
 ``log*`` | log<br>logger | analog
 ``*log`` | analog<br>alog | logg
 ``*log*`` | analog<br>alogger | lop
-``l*g`` | log<br>logg | lop
+``l*g`` | log<br>logg | lake swimming
 
 **Supported**     | **Not Supported**
```
Comment (mckenzielee) [inline Documentation/SequentialDataStore/Searching.md]: This is great.

File: Documentation/SequentialDataStore/Searching.md [hunk 4]
```diff
@@ -184,13 +184,9 @@ Operators | Description
 ``NOT`` | NOT operator. For example, ``cat NOT dog`` searches for streams that have the "cat" term or do not have "dog".  NOT must be in all caps.
 ``*``   | Wildcard operator. For example, ``cat*`` searches for streams that have a term that starts with "cat", ignoring case.
 ``:``   | Field-scoped query.  For example, ``id:stream*`` will search for streams where the ``id`` field starts with "stream", but will not search on other fields like ``name`` or ``description``.  *Note that field names are camel case and are case sensitive.*
+``" "`` | Quote operator. For example, while ``dog food`` (without quotes) would search for streams containing "dog" and/or "food" anywhere in any order, ``"dog food"`` (with quotes) will only match streams that contain the whole string together and in that order.
```
[the remark this answers is not in the document]
Comment (ScottPatterson) [reply Documentation/SequentialDataStore/Searching.md]: I've added a brief description of what the operator does for wildcards, colons, and quotes and fixed the past/present tense confusion.

File: Documentation/SequentialDataStore/Searching.md [hunk 5]
```diff
@@ -184,13 +184,9 @@ Operators | Description
...
 ``( )`` | Precedence operator. For example, ``motel AND (wifi OR luxury)`` searches for streams containing the motel term and either wifi or luxury (or both).
```
[the remark this answers is not in the document]
Comment (ScottPatterson) [reply Documentation/SequentialDataStore/Searching.md]: Fixed.

**Pull request merged**